addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -155,6 +155,13 @@ registerModel({
             }
             this.userSetting.update({ rtcLayout: 'sidebar' });
         },
+        /**
+         * @private
+         * @returns {Object} browser
+         */
+        _computeBrowser() {
+            return this.env.browser;
```

Comment:
Can we already import it instead of accessing it from env, or is it not possible to due missing keys and such?

Reply:
Missing keys, test adaptation. I'd rather do it in the env PR since tests are already heavily remodeled.